packages/dart/noports_core/lib/src/srv/srv.dart

```diff
@@ -32,17 +33,17 @@ abstract class Srv<T> {
   static Srv<Process> exec(
     String host,
     int streamingPort, {
-    required int localPort,
-    required bool bindLocalPort,
+    int? localPort,
+    bool? bindLocalPort,
     String? rvdAuthString,
     String? sessionAESKeyString,
     String? sessionIVString,
   }) {
     return SrvImplExec(
       host,
       streamingPort,
-      localPort: localPort,
-      bindLocalPort: bindLocalPort,
+      localPort: localPort!,
+      bindLocalPort: bindLocalPort!,
```

Comment:
This is an unsafe non-null assertion of localPort and bindLocalPort

Reply:
Ack, thank you. I will clean up those parameters generally